src/cc/arduino/plugins/wifi101/UpdaterImpl.java

```diff
@@ -188,6 +210,8 @@ protected void removeCertificate() {
 
 	@Override
 	protected void uploadCertificates() {
+		Flasher fw = ( Flasher) getFirmwareSelector().getSelectedItem();
```

Comment:
Handling certificates on the NINA will be different from the WINC1500 ... have we considered this?